pkg/apis/scorecard/v1alpha3/configuration_types.go

```diff
@@ -39,5 +40,21 @@ type TestConfiguration struct {
 	// Entrypoint is a list of commands and arguments passed to the test image.
 	Entrypoint []string `json:"entrypoint,omitempty" yaml:"entrypoint,omitempty"`
 	// Labels further describe the test and enable selection.
-	Labels map[string]string `json:"labels,omitempty" yaml:"labels,omitempty"`
+	Labels  map[string]string `json:"labels,omitempty" yaml:"labels,omitempty"`
+	Storage Storage           `json:"storage,omitempty" yaml:"storage,omitempty"`
```

Comment:
Fields and type defs need descriptive godocs, since they are the only docs source for this repo right now.